src/rpc/blockchain.cpp

```diff
@@ -811,6 +812,178 @@ UniValue getblock(const UniValue& params, bool fHelp)
     return blockToJSON(block, pblockindex, verbosity >= 2);
 }
 
+UniValue getblockexpanded(const UniValue& params, bool fHelp)
+{
+    if (fHelp || params.size() < 1 || params.size() > 2)
+        throw runtime_error(
+            "getblock \"hash|height\" ( verbose )\n"
+            "\nIf verbosity is 1, returns an Object with information about the block.\n"
```

Comment:
Can we mention that this command only works if the `-maturityHeightIndex` flag is enabled?